### DISCUSSION TITLE
AVG Blocked copyq 6.4.0 - False Positive?

### DISCUSSION THREAD

**Discussion by urShadoww** [hluk/CopyQ#2240] [General]
I upgraded to copyq 6.4.0 and AVG immediately marked it as infected with Win64:Malwara-gen. This didn't happen with copyq 6.3.2.

![image](https://user-images.githubusercontent.com/19491462/213646038-75f8b3f9-67c5-4b7e-a541-01953cb42c6c.png)

**Comment by hluk** [comment]
Most probably a false positive. :(

Similar to #2239.
